pkg/api/customization/cluster/actionhandler.go

```diff
@@ -56,7 +65,7 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 		}
 		return a.disableMonitoring(actionName, action, apiContext)
 	case v3.ClusterActionBackupEtcd:
-		if !canUpdateCluster() {
+		if !canBackupEtcd() {
 			return httperror.NewAPIError(httperror.Unauthorized, "can not backup etcd")
 		}
 		return a.BackupEtcdHandler(actionName, action, apiContext)
```

Comment:
should restoreFromEtcdBackup be tied to cluster update or etcdbackup mgmt?

Reply:
@cjellick I think cluster update since that will actually change the cluster state?